Cobalt: patch updates to syncup with src tree changes

HaseenaSainul opened the pull request.

MFransen69 reviewed (commented):
Is the rename of the patch file on purpose? Now there are two with 0005

HaseenaSainul commented:
I removed the patch : 0005-changes-for-wayland-sink.patch since it is now available in the cobalt source tree. then I renamed 0006 to 0005 to avoid gap between patches

MFransen69 commented:
Clear, wasn't paying attention very well :)
